src_code/src/routes/InventoryRoutes.js

```diff
@@ -1,435 +1,456 @@
- *          "equipmentCount": 2
- *      },
- *  ]
- * }
- */
-router.get("/models", async(request, response) => {
-    try{ 
-        /** Perform delete models */
-        return await Promise.resolve(inventoryServices.GetAllModels(response));
-    } catch(error) {
-        /** Logging unexpected error. help for debug */
-        console.log("ERROR: There is an error while retrieving model's information:", error);
-        /** Response error message to the client */
-        return responseBuilder.ServerError(response, "There is an error while retrieving model's information.");
-    }
-})
-
-/**
- * GET/RETRIEVE ALL AVAILABLE EQUIPMENTS/MODELS IN A PERIOD OF TIME
- * URL => /api/inventory/available-models?startDate=value&endDate=value
- * Response Body: 
- * {
- *  "message": "Available models successfully retrieved.",
- *  "responseObject": 
- *   [
- *      {
- *          "modelId": 59,
- *          "modelName": "Oakton Aneroid",
- *          "modelPhoto": "https://drive.google.com/uc?id=1NvjXSvJtM1RJQbVAQIqs9NhY4ptF0oUT",
- *          "typeName": "Barometer",
- *          "availableCount": 2
- *      }
- *   ]
- * }
- */
-router.get("/available-models", async(request, response) => {
-    try{ 
-        /** Retrieve values from query parameters */
-        const startDate = request.query.startDate;
-        const endDate = request.query.endDate;
-        /** Perform delete models */
-        return await Promise.resolve(inventoryServices.GetAvailableModels(response, startDate, endDate));
-    } catch(error) {
-        /** Logging unexpected error. help for debug */
-        console.log("ERROR: There is an error while retrieving model's information:", error);
-        /** Response error message to the client */
-        return responseBuilder.ServerError(response, "There is an error while retrieving model's information.");
-    }
-})
-
-/** Exports the router */
-module.exports = router;
+/** Initalize neccessary modules */
+const express = require('express');
+const responseBuilder = require('../utils/interfaces/IResponseBuilder');
+const inventoryServices = require('../controllers/interfaces/IInventory');
+const multer = require("multer");
+
+/** Initialize router for the route */
+const router = express.Router();
+
+/** Set up multer for handling file uploads */
+const storage = multer.memoryStorage();
+const upload = multer({ storage: storage });
+
+/**
+ *  POST/ADD TOOL
+ *  URL => /api/inventory/add
+ */
+router.post('/add', async(request, response) => {
+    try{
+        /** If there is no request body, then we return the request body is empty */
+        if(!request.body || Object.keys(request.body).length === 0) {
+            /** Return request body is empty */
+            return responseBuilder.MissingContent(response, "RB");
+        } 
+        /** If request body is exist, then we perform sign in based on request body */
+        return await Promise.resolve(inventoryServices.AddTool(response, request.body));

```

Comment:
Hmm, I see. I've been getting those two terms confused since our product is titled "ECS Tool Checkout", but we usually use "equipment" when discussing the app. So I guess we stick with "equipment" for dev purposes, and "tool" when communicating with the client or something?